rqd/rqd/cuerqd.py

```diff
@@ -135,137 +108,134 @@ def rebootNow(self):
-            runFrame.log_dir = "/tmp" # This would be on the shottree
-            runFrame.show = "swtest"
-            runFrame.shot = "home"
-            runFrame.uid = 10164
-            runFrame.num_cores = 1
-
-            rqdHost.launchFrame(runFrame)
-
-
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('host', nargs='?', default='localhost', help='RQD hostname (defaults to localhost)')
+    parser.add_argument('-s', action='store_true', help='Print RQD status')
+    parser.add_argument('-v', action='store_true', help='Print RQD version')
+    parser.add_argument('--lp', metavar='coreID', nargs='+', help='Lock the specified cores')
+    parser.add_argument('--ulp', metavar='coreID', nargs='+', help='Unlock the specified cores')
+    parser.add_argument('--lh', action='store_true', help='Lock all cores for the specified host')
+    parser.add_argument('--ulh', action='store_true', help='Unlock all cores for the specified host')
+    parser.add_argument('--nimbyon', action='store_true', help="Turn on 'Not in my back yard' (NIMBY) to stop processing on the specified host")
+    parser.add_argument('--nimbyoff', action='store_true', help="Turn off 'Not in my back yard' (NIMBY) to start processing on the specified host")
+    parser.add_argument('--exit', action='store_true', help='Lock host, wait until machine is idle, and then shutdown RQD. Any unlock command cancels this request.')
+    parser.add_argument('--exit_now', action='store_true', help='KILL ALL running frames and shutdown RQD')
+    parser.add_argument('--restart', action='store_true', help='Lock host, wait until machine is idle, and then restart RQD. Any unlock command cancels this request')
+    parser.add_argument('--restart_now', action='store_true', help='KILL ALL running frames and restart RQD')
+    parser.add_argument('--reboot', action='store_true', help='Lock host, wait until machine is idle, and then REBOOT machine. Any unlock command cancels this request.')
+    parser.add_argument('--reboot_now', action='store_true', help='KILL ALL running frames and REBOOT machine')
+    parser.add_argument('--kill', metavar='frameID', nargs='+', help='Attempts to kill the given frame via its ICE proxy')
+    parser.add_argument('--getproxy', metavar='frameID', nargs='+', help='Returns the proxy for the given frameid')
+    parser.add_argument('--test_edu_frame',action='store_true', help='Launch an edu frame test on an idle core (or first core if none are available)')
+    parser.add_argument('--test_script_frame', action='store_true', help='Launch a script frame test on an idle core (or first core if none are available)')
+    parser.add_argument('--test_script_frame_mac', action='store_true', help='Launch a script frame test for macOS on an idle core (or first core if none are available)')
+    
+    args = parser.parse_args()
+
+    rqdHost = RqdHost(args.host)
+     
+    if args.s:
+        print(rqdHost.status())
+    if args.v:
```

Comment:
nit: Add a blank line before each `if args.` section, to make it a bit easier to read.